mne/preprocessing/tests/test_ica.py

```diff
@@ -383,6 +383,10 @@ def test_ica_additional(method):
 
     plt.close('all')
 
+    ica_different_channels = ICA(n_components=2, random_state=0).fit(
+        raw, picks=[2, 3, 4, 5])
+    pytest.raises(ValueError, corrmap, [ica_different_channels, ica], (0, 0))
```

Comment:
For future reference, the modern way to do this is:
```
with pytest.raises(ValueError, match='whatever'):
    corrmap(...)
```
It makes it immediately clear what the error is (more readable) and helps verify that it's catching the correct error (more precise test). It's possible for example that the `ValueError` here could be raised because the `ICA` was never fit, for example.